src/views/ApartmentApp/components/StudentApplication/components/ApplicantList/components/ApplicantListItem/index.js

```diff
@@ -112,7 +89,11 @@ const ApplicantListItem = ({
         </ListItemAvatar>
         <Grid container alignItems="center" spacing={1}>
           <Grid item xs={8} sm>
-            <ListItemText primary={displayName} secondary={personClass} className={'list-item'} />
+            <ListItemText
+              primary={displayName}
+              secondary={profile.Class ?? ''}
+              className={'list-item'}
```

Comment:
Technically you could pass this as a simple string:
```suggestion
              className="list-item"
```
but it really doesn't matter very much.

Reply:
I have actually been wondering why much of the site uses `className={'example-name'}` instead of `className="example-name"`
I saw it nearly everywhere and didn't understand the distinction back in the fall, so I just assumed there might be some reason

Reply:
This is universally true for the classNames? Because if so, I will do it all at once in all the ApartApp files with a regex statement

Reply:
Technically it's true of all JSX props, yes. If the value you're passing is a string, you can pass a string literal directly: `<Component prop="some string">` is equivalent to `<Component prop={'some string'}>`

Reply:
See the 'Props in JSX' section of this page for more details: https://reactjs.org/docs/jsx-in-depth.html#props-in-jsx

Reply:
That's what I figured based on the meaning of the `{ }` in the JSX content, which is also why I was confused that most things I looked at were consistently using `{'class-name'}`